user-guide/Advanced_Modules/Data_Aggregator/Data_Aggregator_troubleshooting.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: Data_Aggregator_troubleshooting
+---
+
+# Troubleshooting
+
+Check DataAggregator Log Files for Errors
+
+Ensure to inspect the DataAggregator log files for any potential errors. By default, the log file for Data Aggregator is located in: `C:\ProgramData\Skyline Communications\DataMiner DataAggregator\Logs`.
+
+## The DataMiner DataAggregator process is not starting up
+
+When encountering issues with the DataMiner DataAggregator process failing to start, one common reason is an invalid JSON format within the settings. This is often reflected in an event logged in the `Event Viewer`.
```

Comment:
Could write a bit more concisely:
```suggestion
When the DataMiner DataAggregator service fails to start, a common reason is invalid JSON syntax in the settings configuration. In that case, an event will be logged in the `Event Viewer`.
```
Maybe also mention the path or file name of the settings?